README.rst

```diff
@@ -43,6 +43,30 @@ Example Usage
     client = SwaggerClient.from_url("http://petstore.swagger.io/v2/swagger.json")
     pet = client.pet.getPetById(petId=42).result()
 
+Example with Basic Authentication
+-------------------------
+
+.. code-block:: python
+
+    from bravado.requests_client import RequestsClient
+    from bravado.client import SwaggerClient
+    from bravado.swagger_model import load_file
+    
+    http_client = RequestsClient()
+    http_client.set_basic_auth(
+        'api.yourhost.com',
+        'username', 'password'
+    )
+    # some APIs, require explicitly set header "Accept: application/json"
```

Comment:
I am not sure I understand how this is directly related to authentication, should this really be in the default standard example?